packages/core/docs/use-error-handler.md

```diff
@@ -1,19 +1,17 @@
 #### `useErrorHandler`
 
-This plugin triggers a custom function every time execution encounter an error.
```

Comment:
Btw this part didn't reflect the reality, that's why I updated it. I assume the behaviour was changed at some point without updating the docs.